sortbootorder.c

```diff
@@ -223,10 +228,17 @@ static void show_boot_device_list( char buffer[MAX_DEVICES][MAX_LENGTH], u8 line
 	for (i = 0; i < line_cnt; i++ ) {
 		for (y = 0; y < lineDef_cnt; y++) {
 			if (strcmp_printable_char(&(buffer[i][0]), &(bootlist_def[y][0])) == 0) {
-				strcpy(print_device, &bootlist_map[y][0]);
-				print_device[strlen(print_device)-1] = '\0';
-				printf("  %s %s\n", print_device, (device_toggle[y]) ? "" : "(disabled)");
-				break;
+				unique = 1;
+				for (j = 0; j < y; j++) {
+					if (strcmp_printable_char(&bootlist_map[y][0], &bootlist_map[j][0])  ==  0)
+						unique = 0;
+				}
```

Comment:
As far as I know, there still have to be four separate entries in bootorder files. If we want to merge USB options into one in sortbootorder menu then we have to set the same name to all of them and print only one of them. This prevents from printing several items with the same name.